hosts.go

```diff
@@ -398,8 +398,8 @@ func (h *Hosts) removeByPosition(pos int) {
 		h.Clear()
 		return
 	}
-	if pos == len(h.Lines) {
-		h.Lines = h.Lines[:pos-1]
+	if pos == len(h.Lines)-1 {
+		h.Lines = h.Lines[:pos]
 		return
 	}
```

Comment:
Optionally we could just remove the whole block. Previously the condition on line 401 was never true actually and hence, it can also work without it. However, I'm not sure if it's nice to have a slice out of bounds below on 405...